src/move.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
```

Comment:
I would classify the Selection object as a Model object.  The Move and Resize are ViewController objects. The shouldn't be squashed together.